iOS/DuckDuckGo/Subscription/ViewModel/SubscriptionSettingsViewModelV2.swift

```diff
@@ -321,7 +333,24 @@ final class SubscriptionSettingsViewModelV2: ObservableObject {
         }
     }
 
+    func dismissRebrandingMessage() {
+        keyValueStorage.set(true, forKey: bannerDismissedKey)
+        showRebrandingMessage = false
+    }
+
     deinit {
         signOutObserver = nil
     }
 }
+
+public struct SubscriptionSettingsStore: KeyValueStoring {
+
+    private var userDefaults: UserDefaults? { UserDefaults(suiteName: "com.duckduckgo.app.subscriptionSettingsStore") }
```

Comment:
We _could_ use [this](https://app.asana.com/1/137249556945/project/1200194497630846/task/1210288620537332?focus=true) here instead of `UserDefaults`. Not strictly necessary in this scenario (as risk of UD being unavailable is low) but based on the documentation it’s the recommended best practice going forward.